blocks/medium-promo-block/features/medium-promo/default.jsx

```diff
@@ -124,7 +123,7 @@ const MediumPromo = ({ customFields }) => {
     <>
       <article className="container-fluid medium-promo">
         <div className={`medium-promo-wrapper ${customFields.showImage ? 'md-promo-image' : ''}`}>
-          {/* customFields.headlinePosition === 'above'
+          {(customFields.headlinePosition === 'above' || undefined || 'undefined')
```

Comment:
is there an example where this could be string 'undefined' not undefined?

Reply:
I have seen this before and wanted to make sure all the bases were covered.